<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsDialog</string>
  </property>
  <widget class="QTabWidget" name="tabSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Files</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>341</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Shader</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertex-Shader File</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolBtnVertexFile">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditVertexFile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fragment-Shader File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFragFile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolBtnFragFile">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Edge-Blending Images</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolBtnBlendingPath">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditBlendingPath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Visualisation</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>341</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Resolution</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>176</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditResW"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditResH"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushBtnOK">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
